docs/index.md

```diff
@@ -1,47 +1,51 @@
----
+______________________________________________________________________
+
 theme:
-  metaDescription: napari is a fast multi-dimensional image viewer for Python. It can help you **explore** any image-like data, be it 2D, 3D, or even higher-dimensional. It can also help you **overlay** downstream or **associated data**, such as point coordinates or segmentations, which you can use to **annotate** and **proofread** your image data.
-  quickLinks:
-    - title: Community
-      content: Meet the team, our mission, and our values
-      url: /community/index.html
+metaDescription: napari is a fast multi-dimensional image viewer for Python. It can help you **explore** any image-like data, be it 2D, 3D, or even higher-dimensional. It can also help you **overlay** downstream or **associated data**, such as point coordinates or segmentations, which you can use to **annotate** and **proofread** your image data.
+quickLinks:
+\- title: Community
+content: Meet the team, our mission, and our values
+url: /community/index.html
```

Comment:
These changes break the rendering:
https://output.circle-artifacts.com/output/job/8682c349-6e0e-4998-b01c-fe613cf134d4/artifacts/0/docs/_build/index.html

Do we need a myst plugin?

Reply:
Maybe? https://github.com/executablebooks/mdformat-myst